week-3/B-array-some/exercise.js

```diff
@@ -15,6 +16,11 @@ var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 var students = ["Islam", "Lesley", "Harun", "Rukmini"];
 var mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
+function isNull(value) {
+  if (value === null) {
+    process.exit(1);
```

Comment:
This is technically correct, but not quite what the exercise was asking for. Remember that the `.some()` method will return either true (which means that the array does contain some null values) or false (which means that the array does not contain some null values).

We could then use this boolean to decide if we should exit the program.